# Publishes depths and velocities based on the database contents

std_msgs/Header header

# Depths
float32[] depths

# Velocities
geometry_msgs/Vector3[] velocities

